src/Exercise2/ConsumerImplementation.java

```diff
@@ -0,0 +1,19 @@
+package Exercise2;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.function.Consumer;
+
+public class ConsumerImplementation {
+    public static void main(String[] args) {
+
+        final DateTimeFormatter time = DateTimeFormatter.ofPattern("HH:mm");
+        LocalTime now = LocalTime.now();
+
+        Consumer<String> substringTime = (newNow) -> {
+            System.out.println("Godzina: " + newNow.substring(0,2));
```

Comment:
W tym przypadku, zamiast operacji na strinach można używać metody format na formaterze i dzięki temu metoda zrobi to za nas :) nie będziemy musieli zmiennych hardcodować :) 

ZoneId leavingZone = ...;
ZonedDateTime departure = ...;

try {
    DateTimeFormatter format = DateTimeFormatter.ofPattern("MMM d yyyy  hh:mm a");
    String out = departure.format(format);
    System.out.printf("LEAVING:  %s (%s)%n", out, leavingZone);
}
catch (DateTimeException exc) {
    System.out.printf("%s can't be formatted!%n", departure);
    throw exc;
}